modules/statistics/templates/form_stats_behavioural.tpl

```diff
@@ -86,22 +86,22 @@
         <tr>
             <td class="  DD">{$visit|upper}</td>
             {foreach from=$Centers item=center key=centername}
-                <td class='{$center.LongName}DD complete'>{$dde[$center.ID][$visit].complete|default:"0"} ({$dde[$center.ID][$visit].percent|default:"0"}%)</td>
-                <td class='{$center.LongName}DD'>{$dde[$center.ID][$visit].total|default:"0"}</td>
+                <td class='DD complete'>{$dde[$center.ID][$visit].complete|default:"0"} ({$dde[$center.ID][$visit].percent|default:"0"}%)</td>
+                <td class='DD'>{$dde[$center.ID][$visit].total|default:"0"}</td>
             {/foreach}
 
             {/foreach}
         <tr>
             <td class="total">Total</td>
             {foreach from=$Centers item=center key=centername}
-                <td class='{$center.LongName}DD visit_complete total'>{$dde[$center.ID].all.complete|default:"0"} ({$dde[$center.ID].all.percent|default:"0"}%)</td>
-                <td class='{$center.LongName}DD total'>{$dde[$center.ID].all.total|default:"0"}</td>
+                <td class='DD visit_complete total'>{$dde[$center.ID].all.complete|default:"0"} ({$dde[$center.ID].all.percent|default:"0"}%)</td>
+                <td class='DD total'>{$dde[$center.ID].all.total|default:"0"}</td>
             {/foreach}
         </tr>
         <tr>
             <td class=" pis">Per Instrument Stats</td>
             {foreach from=$Centers item=center key=centername}
-                <td id='{$center.LongName}DDPIS' class="pis" colspan="2">
+                <td id=' DDPIS' class="pis" colspan="2">
```

Comment:
CenterIDs aren't valid CSS class names either, since they start with a number and don't have any non-numeric characters.

https://www.w3.org/TR/CSS21/syndata.html#characters
https://www.w3.org/TR/html5/dom.html#the-id-attribute